app/home/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class Playlist(models.Model):
+    owner = models.ForeignKey(User)
+    name = models.CharField(max_length=100)
+    spotify_playlist_id = models.CharField(max_length=255)
```

Comment:
move the `class Song(models.Model)` (also make it singular in name not plural) to the top of the file (in front of Playlist)

then add to Playlist
```
    songs = models.ManyToManyField(Song)
```